requirements.txt

```
@@ -1,4 +1,4 @@
-netCDF4 ~= 1.6.3
-numpy >= 1.24.2,< 2.0
+netCDF4 >= 1.6.3
+numpy >= 1.24.2,<2.3
```

I could change these also to 
```suggestion
netCDF4 >= 1.6.3,<2
numpy >= 1.24.2,<3
```
if you thought that would be better.